chapters/tutorials/tutorial_01.rst

```diff
@@ -0,0 +1,145 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+What is Jina
+----------------------------------
+
+Tutorial
+----------------------------------
+
+At the end of this tutorial, you will have your search engine using text. You will use text as an input and get a matching text for that.
```

Comment:
**Nit**: you will have built a search engine for text data.

Reply:
matching text for that. (For that is very informal)

Perhaps just drop it.

Reply:
That was some bad writing, I re-wrote it.